app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -2,10 +2,31 @@ module Api
   module V1
     module Admin
       class UsersController < ApiController
-        include Api::Admin::UsersController
-
         before_action :authenticate!
         before_action :authorize_super_admin
+
+        def create
+          # NOTE: We can add an inviting user here, e.g. User.invite!(current_user, user_params).
+          User.invite!(user_params)
```

Comment:
This is what you'll find in all v1 controllers: Removing the `include` of the concern and inclusion of the concern code itself in the controller